speech/recognize.js

```diff
@@ -45,6 +45,7 @@ function syncRecognize (filename, encoding, sampleRateHertz, languageCode) {
   // const languageCode = 'en-US';
 
   const config = {
+    enableWordTimeOffsets: false,
```

Comment:
Nit: We prefer not to set optional fields to their default values in the samples.